pkg/pluginmanager/manager.go

```diff
@@ -1091,3 +1095,24 @@ func FindVersion(recommendedPluginVersion, requestedVersion string) string {
 	}
 	return requestedVersion
 }
+
+// getPluginDiscoveries returns the plugin discoveries found in the configuration file.
+func getPluginDiscoveries() ([]configtypes.PluginDiscovery, error) {
+	if configlib.IsFeatureActivated(constants.FeatureCentralRepository) {
+		pd, err := getPreReleasePluginDiscovery()
+		if err != nil {
+			return nil, err
+		}
+		return []configtypes.PluginDiscovery{pd}, nil
+	}
```

Comment:
It is the above block that uses the `TANZU_CLI_PRE_RELEASE_REPO_IMAGE` variable.
If we comment this, we can successfully run the new `Test_InstallPlugin_InstalledPlugins_Central_Repo` unit test.